app/views/users/index.html.erb

```diff
@@ -13,11 +13,9 @@
 
 <% @users.each do |user| %>
   <tr>
-    <td><%= user.name %></td>
+    <td><%= link_to user.name, user_path(user) %></td>
     <td><%= user.email %></td>
-    <td><%= link_to 'Show', user_path(user) %></td>
-    <td><%= link_to 'Edit', edit_user_path(user) %></td>
```

Comment:
Previously, edit also exists in "show". It's redundant to have two links linking to edit view.
if the admin wants to edit the info, he/she usually will click on the user's name to view the info, then decide whether to edit or not. it makes sense to put edit link in show.